tephra-core/src/main/java/co/cask/tephra/util/TxUtils.java

```diff
@@ -30,6 +30,10 @@
  */
 public class TxUtils {
 
+  // Any cell with timestamp lesser than MAX_NON_TX_TIMESTAMP is assumed to be pre-existing data,
+  // i.e. data written before table was converted into transactional table using Tephra.
+  private static final long MAX_NON_TX_TIMESTAMP = (long) (System.currentTimeMillis() * 1.1);
```

Comment:
Why use current time millis to assign to a constant? So the constant change on every run depending on what this class is being loaded?